resources/usa_lottery/server/sv_lottery.lua

```diff
@@ -104,3 +113,23 @@ RegisterServerCallback {
         end
     end
 }
+
+function SaveNewBalance(finishedBalance, fundAccount)
+    TriggerEvent('es:exposeDBFunctions', function(db)
+        db.updateDocument("govfunds", fundAccount, { content = finishedBalance }, function(doc, err, rText)
+            if doc then
+                --print("* Balance updated in DB! *")
+            end
+		end)
+	end)
+end
+
+function GetCurrentBalance(ident, cb, fundAccount)
+    TriggerEvent('es:exposeDBFunctions', function(db)
+        db.getDocumentById("govfunds", ident, function(doc)
```

Comment:
```suggestion
        local doc = exports.essentialmode:getDocumentById("govfunds", ident)
```
Then get rid of the `es:exposeDBFunctions` event part

Cleaner this way